client/ayon_core/plugins/publish/extract_review.py

```diff
@@ -738,10 +809,29 @@ def _ffmpeg_arguments(
                 "-frames:v", str(output_frames_len)
             ])
 
-        # Add video/image input path
-        ffmpeg_input_args.extend([
-            "-i", path_to_subprocess_arg(temp_data["full_input_path"])
-        ])
+        if not explicit_frames:
+            # Add video/image input path
+            ffmpeg_input_args.extend([
+                "-i", path_to_subprocess_arg(temp_data["full_input_path"])
+            ])
+        else:
+            staging_dir = os.path.dirname(temp_data["full_input_path"])
+            explicit_frames_path = os.path.join(
```

Comment:
NOTE: This does create file in staging directory, maybe we should create the file in temp (and remove it when done)?

Reply:
That file gets deleted after ffmpeg is done with it.

Reply:
But inside staging dir.